pywwise/enums.py

```diff
@@ -1302,6 +1305,330 @@ class EImportOperation(_StrEnum):
+	"""The audio format of the media file."""
+	
+	SIZE = "size"
+	"""The size (in bytes) of the media file."""
+	
+	SOUND_BANK = "soundBank"
+	"""The name of the SoundBank that contains the media file."""
+
+
+class EPerformanceMonitorMembers(_StrEnum):
+	"""Enumeration of defined members for a PerformanceMonitorCounter return structure. """
+	
+	NAME = "name"
+	"""Name of the counter as shown in Wwise Authoring."""
+	
+	ID = "id"
+	"""Unique Id of the counter."""
+	
+	VALUE = "value"
+	"""Value of counter at given time."""
+
+
+class EActiveRTPCMembers(_StrEnum):
+	"""Enumeration of defined members for an ActiveRTPCInfo return structure."""
+
+	ID = "id"
+	"""The ID (GUID) of the Game Parameter, LFO, Time, Envelope or MIDI Parameter object."""
+	
+	NAME = "name"
+	"""The name of the Game Parameter, LFO, Time, Envelope or MIDI Parameter object. The name of the object."""
+	
+	GAME_OBJECT_ID = "gameObjectId"
+	"""The Game Object associated with the RTPC scope, or AK_INVALID_GAME_OBJECT for global scope RTPCs."""
+	
+	VALUE = "value"
+	"""The value of the Game Parameter, LFO, Time, Envelope or MIDI Parameter at the cursor time."""
+
+
+class EVoicePipelineReturnOptions(_StrEnum):
+	"""Enumeration of defined members for a voice pipeline return structure."""
+	
+	PIPELINE_ID = "pipelineID"
+	"""Pipeline ID of the voice."""
+	
+	PLAYING_ID = "playingID"
+	"""Playing ID of the voice."""
+	
+	SOUND_ID = "soundID"
+	"""Short ID of the sound object corresponding to the voice."""
+	
+	GAME_OBJECT_ID = "gameObjectID"
+	"""Game Object ID corresponding to the voice."""
+	
+	GAME_OBJECT_NAME = "gameObjectName"
+	"""Game Object Name corresponding to the voice."""
+	
+	OBJECT_GUID = "objectGUID"
+	"""Object GUID corresponding to the voice."""
+	
+	OBJECT_NAME = "objectName"
+	"""Object Name corresponding to the voice."""
+	
+	PLAY_TARGET_ID = "playTargetID"
+	"""Short ID of the play target object corresponding to the voice."""
+	
+	PLAY_TARGET_GUID = "playTargetGUID"
+	"""GUID of the play target object corresponding to the voice."""
+	
+	PLAY_TARGET_NAME = "playTargetName"
+	"""Name of the play target object corresponding to the voice."""
+	
+	BASE_VOLUME = "baseVolume"
+	"""Voice volume in dB, including HDR attenuation."""
+	
+	GAME_AUX_SEND_VOLUME = "gameAuxSendVolume"
+	"""Volume send to the auxiliary bus send in dB."""
+	
+	Envelope = "envelope"
+	"""Current analyzed envelope value in dB. 0 if unavailable."""
+	
+	NORMALIZATION_GAIN = "normalizationGain"
+	"""Loudness normalization and make-up gain in dB."""
+	
+	LOW_PASS_FILTER = "lowPassFilter"
+	"""Low-Pass Filter applied to the voice."""
+	
+	HIGH_PASS_FILTER = "highPassFilter"
+	"""High-Pass Filter applied to the voice."""
+	
+	Priority = "priority"
+	"""Priority given to the voice."""
+	
+	IS_STARTED = "isStarted"
+	"""If the voice has started."""
+	
+	IS_VIRTUAL = "isVirtual"
+	"""If the voice is virtual."""
+	
+	IS_FORCED_VIRTUAL = "isForcedVirtual"
+	"""If the voice was forced virtual."""
```

Comment:
The member `Envelope` is not following the ALL_CAPS naming convention. Should be `ENVELOPE`.